channels/views_test.py

```diff
@@ -187,7 +187,7 @@ def test_patch_channel_image(client, channel, attribute):
         os.path.dirname(__file__),  # noqa: PTH120
         "..",
         "frontends",
-        "mit-learn",
+        "main",
         "public",
         "images",
         "blank.png",
```

Comment:
I've added a blank.png at this path, but do we actually need it for this test? It's not used in the frontend.

Reply:
It's "supposed" to be the fallback for embedly. See frontends/ol-utilities/src/learning-resources/learning-resources.ts

That said:
- we don't need the /static version
- The version we do have set isn't working right now (https://rc.learn.mit.edu/static/images/blank.png) ... unsure why